### ISSUE TITLE
icon with vue-fontawesome comes up with grey

### ISSUE THREAD

**Issue by SnowNooDLe** [yaminncco/vue-sidebar-menu#110]
Hi all,
From this issue, #52 
I found the way of using vue-fontawesome.
So I did put 

```JavaScript
// inside main.ts
import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
Vue.component('font-awesome-icon', FontAwesomeIcon);
```

```JavaScript
// inside component
<script lang="ts">
import { library } from "@fortawesome/fontawesome-svg-core";
import { faUserSecret } from "@fortawesome/free-solid-svg-icons";
library.add(faUserSecret);
```
and for menu
```JavaScript
menu = [{
       href: 'path',
       title: "Referral List",
       icon: {
           element: "font-awesome-icon",
           attributes: {
              icon: "user-secret"
           }
       }      
}]
```
But I am getting this.

![Screen Shot 2020-01-10 at 1 37 36 PM](https://user-images.githubusercontent.com/7849113/72116213-6c265900-33ae-11ea-9600-275c09685340.png)

As you can see, for some reasons it comes up with a grey icon instead of an actual icon,
am I missing something?

**Comment by SnowNooDLe** [comment]
Also, tested with example pages, also getting the same grey icon.
Changing menu like what I found from other issues and seems should work.
<img width="365" alt="Screen Shot 2020-01-10 at 5 21 40 PM" src="https://user-images.githubusercontent.com/7849113/72125513-c4b91e80-33cd-11ea-920a-6fb8b144a5ac.png">
Hence example using Font Awesome 5 free? I used Vue-fontawesome, so installed and imported to use it.
<img width="568" alt="Screen Shot 2020-01-10 at 5 21 51 PM" src="https://user-images.githubusercontent.com/7849113/72125539-d5699480-33cd-11ea-846f-c368f3762bee.png">
However, unlike the other answers, with example page also coming up with grey icon, could this be a bug then?
![Screen Shot 2020-01-10 at 5 21 57 PM](https://user-images.githubusercontent.com/7849113/72125603-09dd5080-33ce-11ea-8798-1e732a08edbe.png)
